### ISSUE TITLE
0.5.6 using the older image eugenmayer/unison:2.51.2.0 - monit not working

### ISSUE THREAD

**Issue by rodrigoaguilera** [EugenMayer/docker-sync#562]
### Error
I enabled monit because everyday I trigger the condition were the files stop syncing.
After a while of seein the CPU spiking way above 100% I checked `docker stats` to realize that myapp-sync was eating all the CPU.

I logged into the sync container with the intention of restarting it manually the same way monit does.
docker exec -it myapp-sync /bin/bash

I checked /etc/monitrc and I noticed that the supervisorctl commands reference a process named "unison" and /etc/supervisor.conf.d/supervisor.daemon.conf use "unison-hostsync".

I checked the image repo 
https://github.com/EugenMayer/docker-image-unison
And I noticed this already fixed on the tag 2.51.2.1 but docker-sync is pulling 'eugenmayer/unison:2.51.2.0'

I think this is stopping monit from working.

### Docker Driver
d4m 

### Sync strategy
native_osx

### your docker-sync.yml
```
version: '2'

syncs:
  # IMPORTANT: This name must be unique and should NOT match your real
  # application container name!
  myapp-sync:
    src: './'
    sync_userid: '33'
    sync_excludes: ['vendor/', '.git/', '.sass-cache/', 'sass-cache/', 'node_modules/']
    monit_enable: true
```

### OS
MacOS sierra 10.12.6

**Comment by EugenMayer** [comment]
@rodrigoaguilera thanks for digging for that, did just miss that. i guess that a call for 0.5.7, including 

https://github.com/EugenMayer/docker-sync/issues/555
https://github.com/EugenMayer/docker-sync/issues/554
and also this will fix https://github.com/EugenMayer/docker-sync/issues/553

**Comment by EugenMayer** [comment]
will be fixed by 0.5.7 just right now
